packages/core/src/_overrides.scss

```diff
@@ -9,6 +9,15 @@
   text-transform: uppercase;
 }
 
+.#{$ns}-button:focus {
+  outline: none;
+}
+
+.#{$ns}-button-text {
```

Comment:
Are we trying to override something with `!important` ... otherwise I would go with more specificity.